apps/consent-ui/src/config/appConfig.ts

```diff
@@ -19,7 +19,7 @@
 
 import 'server-only';
 
-import { defaultAppConfig, AppConfig } from 'src/config/types';
+import { defaultAppConfig, AppConfig } from './types';
```

Comment:
Let's revert this back to the full path just for consistency with the clientConfig:
```suggestion
import { defaultAppConfig, AppConfig } from 'src/config/types';
```